src/test/java/org/sciborgs1155/robot/DriveTest.java

```diff
@@ -0,0 +1,47 @@
+package org.sciborgs1155.robot;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.sciborgs1155.lib.TestingUtil.*;
+
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.sciborgs1155.robot.Drive.Drive;
+
+public class DriveTest {
+    static final double DELTA = 1;
+
+    Drive drive;
+    @BeforeEach
+    public void before() {
+        drive = new Drive();
```

Comment:
before you make the drive, setup HAL using the setupHAL() method in TestingUtil

Reply:
Also, can you call this setup or something instead of before?

Reply:
got it